src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -65,14 +65,20 @@ export const ProfilePictureInput: React.FunctionComponent<ProfilePictureInputPro
 
   const hasError = !!error;
 
+  const { src: pictureSrc, onError: onPictureSrcError } = determineAvatar({
+    pictureUrl,
+    userInfo: user,
+  });
+
   return (
     <div className="ProfilePictureUploadForm">
       <div
         className="ProfilePicturePreviewContainer"
         onClick={() => uploadRef.current?.click()}
       >
         <img
-          src={determineAvatar({ pictureUrl, userInfo: user })}
+          src={pictureSrc}
+          onError={onPictureSrcError}
```

Comment:
Interesting enough, we can also try to be clever and use spread `{ ... determineAvatar() }` here... But current code is also good, it's easy to follow.